scripts/dump_saver.bash

```diff
@@ -0,0 +1,52 @@
+echo "programm start..."
```

Comment:
Rename message as it's not a program. Smth like "script for saving dumps starts.."